notebook/notebookapp.py

```diff
@@ -969,7 +972,7 @@ def _notebook_dir_validate(self, proposal):
             # If we receive a non-absolute path, make it absolute.
             value = os.path.abspath(value)
         if not os.path.isdir(value):
-            raise TraitError("No such notebook dir: %r" % value)
+            raise TraitError(_("No such notebook dir: %r") % value)
```

Comment:
Unfortunately here the convention of using `_` for translatable strings has clashed with the Python convention of using `_` for a variable that won't be used.